tensorboard/util/util.py

```diff
@@ -63,6 +65,9 @@ def setup_logging(streams=(sys.stderr,)):
   werkzeug_logger = logging.getLogger('werkzeug')
   werkzeug_logger.setLevel(logging.WARNING)
   werkzeug_logger.handlers = handlers
+  absl_logger = logging.getLogger('absl')
```

Comment:
@nfelt should we just format everything absl or should we change the code so we use tensorboard logger?

Reply:
I'm not 100% sure I understand the question, but now that you mention it, it probably would be a good idea to use specific logger instances (like tensorflow uses theirs) rather than the root logger, which makes our logging better behaved if it's embedded into other programs.

We could either just do something like this:
```py
log = logging.getLogger('tensorboard')  # or logging.getLogger(__name__)
...
def foo():
  log.info(...)
```

Or we could define something like a tensorboard.util.logging module, with a get_logger() function that returns the TB logger.  With that arrangement we could just put the fake absl dependency on that module, and most of our code would depend on that module instead.  (We could also move this function there.)